content/doc/book/installing/_setup-wizard.adoc

```diff
@@ -68,8 +68,8 @@ NOTE: If you are not sure what plugins you need, choose **Install suggested
 plugins**.
 You can install (or remove) additional Jenkins plugins at a later point in time
 via the
-link:../../book/managing[**Manage Jenkins**] >
-link:../../book/managing/plugins/[**Manage Plugins**] page in Jenkins.
+link:../book/managing[**Manage Jenkins**] >
+link:../book/managing/plugins/[**Manage Plugins**] page in Jenkins.
```

Comment:
Since this file is included in other files, wouldn't it be safer to use site relative URL's rather than path relative URL's?
```suggestion
link:/doc/book/managing[**Manage Jenkins**] >
link:/doc/book/managing/plugins/[**Manage Plugins**] page in Jenkins.
```